### DISCUSSION TITLE
Having too many messages in the queue, how to force it to resend?

### DISCUSSION THREAD

**Discussion by didoMova** [postalserver/postal#1972] [Help with using Postal]
I'm trying to force sending 40 000 emails for the queue.
Any suggestions on how to send it?

**Comment by willpower232** [comment]
What problem are you having?

**Reply by didoMova** [reply]
My emails are stuck in a queue.
I use Mautic as an application on other VPS.
It successfully sends about 20k mails and the other 40k mails are in the queue. 

1. I want to know, why did that happen?
2. How to force sending queue mails?
Thank you!

PS 
VPS for Postal is 4 vCPU and 8gb ram.

**Reply by didoMova** [reply]
> What problem are you having?

If you need any other pieces of information, I'll let you know.
:)

**Reply by willpower232** [reply]
what is your `postal status`?

**Reply by didoMova** [reply]
Name                     Command               State   Ports
Here is the status:
------------------------------------------------------------------
postal_cron_1       /docker-entrypoint.sh post ...   Up           
postal_requeuer_1   /docker-entrypoint.sh post ...   Up           
postal_smtp_1       /docker-entrypoint.sh post ...   Up           
postal_web_1        /docker-entrypoint.sh post ...   Up           
postal_worker_1     /docker-entrypoint.sh post ...   Up   


Also, I tried to add about 10-50 workers, didn't change anything.

**Reply by willpower232** [reply]
Did you use the scale command like this? https://github.com/postalserver/postal/discussions/1907
